srcpkgs/golang-migrate/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'golang-migrate'
+pkgname=golang-migrate
```

Comment:
per [guidelines](https://github.com/void-linux/void-packages/blob/master/Manual.md#programs), use the repo name, `migrate`

Reply:
My bad! I will update this